### PR TITLE
base32: add Turkish translation

### PR DESCRIPTION
**Pull request by Yutyo**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by bl-ue — CHANGES_REQUESTED** [review]
Looks good. I'd like to wait on merging this, though, because we have a PR in progress (#5234) that's modifying `base64.md` and `base32.md` and I'd like to get that merged and this updated first.

File: pages.tr/common/base32.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+> Bir dosya veya standart veriyi Base32 formatında şifrele veya yalın veri çıktısı olarak deşifre et.
+
+- Bir dosyayı şifrele:
+
+`base32 {{dosyaismi}}`
+
+- Bir dosyayı deşifre et:
+
+`base32 -d {{dosyaismi}}`
```
Comment (bl-ue) [inline pages.tr/common/base32.md:11]:
```suggestion
`base32 --decode {{dosyaismi}}`
```

File: pages.tr/common/base32.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- stdin'den şifrele:
+
+`{{herhangibirkomut}} | base32`
+
+- stdin'den deşifre et:
+
+`{{herhangibirkomut}} | base32 -d`
```
Comment (bl-ue) [inline pages.tr/common/base32.md:19]:
```suggestion
`{{herhangibirkomut}} | base32 --decode`
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
@Yutyo, please apply these changes from #5234.

**Review by bl-ue — APPROVED** [review]

File: pages.tr/common/base32.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# base32
+
+> Bir dosya veya standart veriyi Base32 formatında şifrele veya yalın veri çıktısı olarak deşifre et.
+
+- Bir dosyayı şifrele:
```
Comment (navarroaxel) [inline pages.tr/common/base32.md:5]: I think you're using the verb `encrypt` (`şifrele`) but the correct verbs are encode and decode.

Comment (bl-ue) [reply pages.tr/common/base32.md:5]: Yeah, maybe.  Google translates `encode` to `kodlamak` 🤔

Comment (bl-ue) [reply pages.tr/common/base32.md:5]: Well, Turkish isn't just a word-to-word mapping of English.  There's likely lots of other, subtle differences.  Let's leave it as is 👍🏻

**Pull request merged**